tests/ColumnSchemaBuilderTest.php

```diff
@@ -2,30 +2,31 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Db\Tests\Mysql;
+namespace Yiisoft\Db\Mysql\Tests;
 
 use Yiisoft\Db\Mysql\ColumnSchemaBuilder;
-use Yiisoft\Db\Schemas\Schema;
+use Yiisoft\Db\Mysql\Schema;
 use Yiisoft\Db\Tests\ColumnSchemaBuilderTest as AbstractColumnSchemaBuilderTest;
 
-final class ColumnSchemaBuilderTest extends AbstractColumnSchemaBuilderTest
+class ColumnSchemaBuilderTest extends AbstractColumnSchemaBuilderTest
```

Comment:
Why did you delete `final`?

Reply:
Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue yiisoft/db#108.